packages/packages/documents/src/hooks/__tests__/use-document-preview-props.test.ts

```diff
@@ -0,0 +1,40 @@
+import { renderHook } from '@testing-library/react-hooks';
+import useDocumentPreviewProps from '../use-document-preview-props';
+import { runCommand } from '@elementor/v1-adapters';
+
+const mockDocument = {
+	id: 1,
+	title: 'Document 1',
+	status: 'publish' as const,
+	isDirty: false,
+	isSaving: false,
+	isSavingDraft: false,
+	userCan: {
+		publish: true,
+	},
+};
+
+jest.mock( '../use-active-document.ts', () => ( {
+	__esModule: true,
+	default: jest.fn( () => mockDocument ),
```

Comment:
Maybe use the `createMockDocument` util?

Reply:
Done.